### PR TITLE
fix(site): pass correct code language to codesandbox

### PR DESCRIPTION
**Pull request by holazz**

### CODE REVIEW

**Review by amadeus711 — APPROVED** [review]

File: demo/utils/EditInCodeSandboxButton.vue [hunk 1]
```diff
@@ -47,8 +47,20 @@ export default defineComponent({
...
+
+    watch(
+      () => props.code,
+      (value) => {
+        parameters.value = getCodeSandboxParams(value)
+      },
+      {
+        immediate: true
+      }
+    )
```
Comment (07akioni) [inline demo/utils/EditInCodeSandboxButton.vue:60]: We should use computed, not a watch like this

Comment (holazz) [reply demo/utils/EditInCodeSandboxButton.vue:60]: You're right, I forget it.😅

File: demo/utils/EditInCodeSandboxButton.vue [hunk 2]
```diff
@@ -47,8 +47,9 @@ export default defineComponent({
     size: String
   },
   setup (props) {
+    const parameters = computed(() => getCodeSandboxParams(props.code))
     return {
-      parameters: getCodeSandboxParams(props.code)
+      parameters
```
Comment (07akioni) [inline demo/utils/EditInCodeSandboxButton.vue:52]:
```suggestion
      parameters: computed(() => getCodeSandboxParams(props.code))
```

File: demo/utils/EditInCodeSandboxButton.vue [hunk 3]
```diff
@@ -47,8 +47,9 @@ export default defineComponent({
     size: String
   },
   setup (props) {
+    const parameters = computed(() => getCodeSandboxParams(props.code))
```
Comment (07akioni) [inline demo/utils/EditInCodeSandboxButton.vue:50]:
```suggestion
```

**Pull request merged**